sherpa/astro/data.py

```diff
@@ -3799,22 +3796,41 @@ def get_filter(self, group=True, format='%.12f', delim=':'):
 
         >>> pha.set_analysis('energy')
         >>> pha.notice(0.5, 7)
-        >>> pha.get_filter(format=%.4f')
-        ''0.5183:8.2198''
+        >>> pha.get_filter(format='%.4f')
+        '0.4672:9.8696'
         >>> pha.set_analysis('channel')
         >>> pha.get_filter()
-        '36:563'
-
-        The default is to show the data range for the grouped
-        dataset, which uses the center of each group. If
-        the grouping is turned off then the center of the
-        start and ending channel of each group is used
-        (and so show a larger data range):
-
-        >>> pha.get_filter(format=%.4f')
-        '0.5183:8.2198'
-        >>> pha.get_filter(group=False, format=%.4f')
-        '0.4745:9.8623'
+        '33:676'
+
+        The filter expression shows the first selected channel to the
+        last one, and so is independent of whether the data is grouped
+        or not:
```

Comment:
I don't like that wording. Below, you show "the selected range does depend on whether the data is grouped or not". It can't be, at the same time, depend and not depend on grouping. Maybe "the group argument to get_filter does not change the filter expression (it is only here for consistency with other functions), but grouping and ungrouping the data *does* change the selected range"  ?

Reply:
Unfortunately I don't have time to make changes other than simple typo fixes. This behavior is related to #1309 - which I realized from writing this PR - and I claim that at the moment you can have different behavior because it's technically a different meaning of "group" (the question is should we, and fixing this would be an actual code change which we can't get in before the release, as it isn't in a critical code path). 

For users of sherpa.astro.ui it doesn't matter as the `get_filter` call there does not have a `group` argument. This is only relevant for users of the `sherpa.astro.data.DataPHA.get_filter` call.